### ISSUE TITLE
Add issuance of debt to cashflow statement

### ISSUE THREAD

**Issue by kpad-git** [dpguthrie/yahooquery#70]
The cash_flow function provides a column for "RepaymentOfDebt", but this value is not net of issuance (it only adds up all the debt payments even if those payments are offset by new debt issuance). Is it possible to add an "IssuanceOfDebt" column to allow computations of net values?

**Comment by dpguthrie** [comment]
Is that value available through YF?

**Comment by kpad-git** [comment]
Yes it is. For example, look at the 12/31/2020 cashflow statement for AT&T (
https://finance.yahoo.com/quote/T/cash-flow?p=T). There are six relevant
values:

1) Net Long Term Debt Issuance:  -7,976,000 (this is the sum of the next
two)
2) Long Term Debt Issuance:  31,988,000
3) Long Term Debt Payments:  -39,964,000
4) Net Short Term Debt Issuance: -44,000 (this is the sum of the next two)
5) Short Term Debt Issuance:  9,440,000
6) Short Term Debt Payments:    -9,484,000

Right now, yahooquery returns -4.9448e10 for "RepaymentOfDebt", which is
the sum of #3 and #6. If you can provide #2 plus #5 as an "IssuanceOfDebt",
it would allow proper calculations. If it's easy to provide all 6, that
would be even better.

Thanks for the awesome work - this API really works beautifully!

**Comment by dpguthrie** [comment]
Should be available in 2.2.15:

```
from yahooquery import Ticker

t = Ticker('T')
df = t.cash_flow()
df.columns
Index(['asOfDate', 'periodType', 'currencyCode', 'AssetImpairmentCharge',
       'BeginningCashPosition', 'CapitalExpenditure',
       'CapitalExpenditureReported', 'CashDividendsPaid',
       'CashFlowFromContinuingFinancingActivities',
       'CashFlowFromContinuingInvestingActivities',
       'CashFlowFromContinuingOperatingActivities',
       'ChangeInCashSupplementalAsReported', 'ChangeInOtherCurrentAssets',
       'ChangeInOtherWorkingCapital', 'ChangeInPayablesAndAccruedExpense',
       'ChangeInReceivables', 'ChangeInWorkingCapital',
       'ChangesInAccountReceivables', 'ChangesInCash',
       'CommonStockDividendPaid', 'CommonStockIssuance', 'CommonStockPayments',
       'DeferredIncomeTax', 'DeferredTax', 'DepreciationAmortizationDepletion',
       'DepreciationAndAmortization', 'EarningsLossesFromEquityInvestments',
       'EndCashPosition', 'FinancingCashFlow', 'FreeCashFlow',
       'GainLossOnInvestmentSecurities', 'IncomeTaxPaidSupplementalData',
       'InterestPaidSupplementalData', 'InterestReceivedCFI',
       'InvestingCashFlow', 'IssuanceOfCapitalStock', **'IssuanceOfDebt'**,
       'LongTermDebtIssuance', 'LongTermDebtPayments',
       'NetBusinessPurchaseAndSale', 'NetCommonStockIssuance', 'NetIncome',
       'NetIncomeFromContinuingOperations', 'NetInvestmentPurchaseAndSale',
       'NetIssuancePaymentsOfDebt', 'NetLongTermDebtIssuance',
       'NetOtherFinancingCharges', 'NetOtherInvestingChanges',
       'NetPPEPurchaseAndSale', 'NetPreferredStockIssuance',
       'NetShortTermDebtIssuance', 'OperatingCashFlow', 'OperatingGainsLosses',
       'OtherNonCashItems', 'PensionAndEmployeeBenefitExpense',
       'PreferredStockIssuance', 'ProvisionandWriteOffofAssets',
       'PurchaseOfBusiness', 'PurchaseOfInvestment', 'PurchaseOfPPE',
       'RepaymentOfDebt', 'RepurchaseOfCapitalStock', 'SaleOfBusiness',
       'ShortTermDebtIssuance', 'ShortTermDebtPayments'],
      dtype='object')

**Comment by kpad-git** [comment]
Wow, awesome... you are a wizard, and a fast one at that!
